test/testUtils/fixturesUtils.py

```diff
@@ -19,9 +19,6 @@
 import sys
 import unittest
 
-# Plugins that are bundled and loaded by default in a Maya installation
```

Comment:
ArubaTesselator plugin is unused by our tests, and modelingToolkit was only used by a single test.